cwp/cwp-wg-papers.md

```diff
@@ -22,16 +22,52 @@ recommended.
     - Download your Google Doc as docx, a.k.a. MS Word, format
     - Using [pandoc](https://pandoc.org/), which is easily [installed](https://pandoc.org/installing.html),
       convert your docx file to latex:
-        - `pandoc -f docx -t latex -o wgpaper.tex wgpaper.docx`
+        ```bash
+        pandoc -f docx -t latex -o wgpaper.tex wgpaper.docx
+        ```
     - The conversion is generally very accurate, but within `itemize` environments you
       may find that text is inside a spurious `quote` environment and that some
       odd non-breaking spaces appear. Just delete these.
-1. Adopt the JHEP preprint style, using the template in the
+1. Adopt the [JHEP preprint](https://jhep.sissa.it/jhep/help/JHEP/TeXclass/DOCS/JHEP-author-manual.pdf)
+   style, using the template in the
    [software development paper](https://github.com/HSF/documents/tree/master/CWP/papers/HSF-CWP-2017-13_soft-dev/latex/).
     - Copy the `jheppub.sty` file into your latex area as it is needed for arXiv side compilation.
-1. The paper title should be `HEP Software Foundation Community White Paper Working Group \-- *My Working Group*`.
-1. You will need to add an abstract for arXiv.
-1. Add a copyright and license statement as a comment - we strongly recommend <br>`% Copyright (C) 2018, The HSF Community White Paper authors, licence CC-BY-4.0.`
+1. Build the author list: the **strongly recommended** way is to create a text file 
+   `authors.txt` and use the `a2tex.py` tool to generate the TeX-formatted author list, 
+   following the steps below:
+   * `authors.txt`: the easiest is to start from the global CWP 
+     [author list](https://github.com/HSF/documents/tree/master/CWP/papers/HSF-CWP-2017-01_roadmap/authors/authors.txt)
+     and remove unnecessary entries. The line format in this file is:
+       ```
+       Surname, Forename - Affiliation_ID (footnote_key)
+       ```
+   * In the previous line, `Affiliation_ID` refers to the first word in the
+   [affiliation list](https://github.com/HSF/documents/tree/master/CWP/papers/HSF-CWP-2017-01_roadmap/authors/addresses.txt) 
+   and `footnote_key` to the number at the start of the line in the
+   [footnote list](https://github.com/HSF/documents/tree/master/CWP/papers/HSF-CWP-2017-01_roadmap/authors/footnotes.txt). 
+   If there are several affiliations, use ` & ` as the separator. If there are several 
+   footnotes, separate them with a comma without surrounding spaces.
+
+1. To create the title page use the `\maketitle` statement (without arguments). **Before** 
+you need to define the title, the abstract and the author list:
+  * The title must be `HEP Software Foundation Community White Paper Working Group \-- *My Working Group*`. 
```

Comment:
I just noticed this is a little weird. Should remove the backslash and the asterisks.